zebra/zebra_vty.c

```diff
@@ -712,6 +726,7 @@ DEFPY(ip_route_blackhole,
 	  |(1-255)$distance                                                   \
 	  |vrf NAME                                                           \
 	  |label WORD                                                         \
+	  |table (1-4294967295)                                               \
```

Comment:
This needs to be a separate defun so that you can install the table keyword for multicast mode.

Reply:
I had done it for vrf node commands, but was thinking that it was ok for default vrf. so I updated it.